packages/docs/docs/careplans/tasks/tasks.md

```diff
@@ -0,0 +1,140 @@
+---
+tags:
+  - care plans
+  - workflow
+  - tasks
+---
+
+# Using `Tasks` to Manage Workflow
+
+## Introduction
+
+Workflow is an essential part of healthcare, and healthcare operations requiring coordination of many manual steps physicians, patients, nurses, care coordinators, etc.
+
+While the majority of FHIR resources represent clinical data that is _operated on_, FHIR also defines a set of workflow resources that describe and track _work to be done._ This guide will discuss the usage of the [`Task`](/docs/api/fhir/resources/task) resource, which is the basic building block resource for tracking workflow progress.
```

Comment:
Consider mentioning here that Tasks are in care plan section as they can comprise elements of a care plan.  A specific example might be Task could represent ensuring that a practitioner completes a PHQ-9 for a patient as part of their onboarding.